test/e2e/failure_domain_lifecycle_test.go

```
@@ -0,0 +1,466 @@
+	} else {
+		t.Log("OrphanCleanupPending condition cleared — orphan tag cleanup succeeded")
+	}
+}
+
+// verifyOrphanConditionPending checks that the OrphanCleanupPending condition is True.
+func verifyOrphanConditionPending(t *testing.T, ctx context.Context, configClient *configclient.Clientset) {
+	t.Helper()
+	err := wait.PollUntilContextTimeout(ctx, failureDomainPollInterval, 2*time.Minute, false, func(pollCtx context.Context) (bool, error) {
+		co, err := configClient.ConfigV1().ClusterOperators().Get(pollCtx, storageCOName, metav1.GetOptions{})
+		if err != nil {
+			return false, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 385) but it returns nil

(nilerr)